wizeline/wize-q-front
# Feat - WIP 247 - Proof of Anonymity

### What does this PR do?

Add a dropdown menu to select between Anonymous or Username, if the Username is selected a pop-up is shown when submitting the question, warning the user that the username will be posted, and asking for a confirmation/cancel.

### Where should the reviewer start?

src/components/QuestionInput.js on the render function of the class, the Dropdown button was added and the SubmitWithUserModal modal was added.
src/Auth.js a function was added to get the profile's name, this function is used in QuestionInput.js.

### How should this be manually tested?
Once logged in, the user should be able to see a dropdown before the input field for the question, this dropdown has Anonymous as default and its value can be changed when clicked. If the username is selected and the user clicks the "Ask Question!" button, a pop-up should be shown, asking for a confirmation, if it is cancelled, the dropdown value and the question input are not reset and the question is not submitted, if the user submits the question, the values are reset to Anonymous, and the inputfield is reset to empty ' '. 

### What are the relevant tickets?
[WIP - 247 [FronEnd] Proof of Anonymity](https://wizeline.atlassian.net/browse/WIP-247)

### Screenshots
![image](https://user-images.githubusercontent.com/15947670/27457908-1417d318-576c-11e7-9c51-bd87f5a2205e.png)
![image](https://user-images.githubusercontent.com/15947670/27457939-2af70608-576c-11e7-8075-3d2de55e8e2c.png)
